tests/microos/install_image.pm

```diff
@@ -17,13 +17,29 @@ use Utils::Architectures 'is_x86_64';
 use power_action_utils 'power_action';
 use serial_terminal 'select_serial_terminal';
 
+sub get_disk_by_wwn {
+    my $wwn = shift;
+    $wwn =~ s/^wwn-//;
+
+    my $name;
+    my $output = script_output('lsblk -d -o name,wwn');
+    for my $line (split /\n/, $output) {
+        if ($line =~ /\Q$wwn\E$/) {
+            ($name) = split(/\s+/, $line);
+        }
+    }
+    die "WWN ${wwn} not found in\n${output}" unless $name;
+    return $name;
```

Comment:
The `return` should be in the loop. Then the error doesn't need a condition. Or do you expect multiple disks with the same WWN and you want to return the last one?